sickbeard/webserve.py

```diff
@@ -3643,6 +3643,16 @@ def saveGeneral(self, log_dir=None, log_nr = 5, log_size = 1048576, web_port=Non
         sickbeard.DOWNLOAD_URL = download_url
         sickbeard.INDEXER_DEFAULT_LANGUAGE = indexerDefaultLang
         sickbeard.LAUNCH_BROWSER = config.checkbox_to_value(launch_browser)
+        if sickbeard.SHOWUPDATE_HOUR != config.to_int(showupdate_hour):
+            sickbeard.showUpdateScheduler.stop.set()
+            logger.log(u"Waiting for the SHOWUPDATER thread to exit so we can set new start hour")
+            while sickbeard.showUpdateScheduler.action.amActive:
+                time.sleep(3)
+            logger.log(u"Starting SHOWUPDATER thread with the new start hour")
+            showUpdateScheduler = scheduler.Scheduler(showUpdater.ShowUpdater(),
```

Comment:
It should be like this: sickbeard.showUpdateScheduler = scheduler.Scheduler(showUpdater.ShowUpdater(),